pkg/proc/proc.go

```diff
@@ -158,6 +161,7 @@ func Continue(dbp Process) error {
 					return err
 				}
 			} else {
+				curthread.Common().returnValues = curbp.Breakpoint.returnInfo.Collect(curthread)
```

Comment:
`curbp.Breakpoint.returnInfo` is a pointer and _could_ be nil here. I suppose we could always assume an internal non-step breakpoint must always have `returnInfo` set, however we should be more intentional about this and check for the pointer being nil and panic with a better error message, since this would be a programmer error.

Alternatively, we could ignore this if there is no `returnInfo` set.

Reply:
`(*returnBreakpointInfo).Collect` works fine on nil pointers.